### PR TITLE
spt's nits

### PR DESCRIPTION
**Pull request by seanturner**
Just some nits.

### CODE REVIEW

File: draft-ietf-tls-subcerts.md [hunk 1]
```diff
@@ -153,14 +161,14 @@ validity period longer than 7 days.  This mechanism is described in detail in
...
-4.2.1.3)  In order to prevent this type of cross-protocol attack, we define a
+4.2.1.3).  In order to prevent this type of cross-protocol attack, we define a
 new DelegationUsage extension to X.509 that permits use of delegated
 credentials.  (See {{certificate-requirements}}.)
 
 ## Rationale
 
 Delegated credentials present a better alternative than other delegation
-mechanisms like proxy certificates {{RFC3820}} for several reasons:
+mechanisms like proxy certificates {{?RFC3820}} for several reasons:
```
Comment (cjpatton) [inline draft-ietf-tls-subcerts.md]: What's this question mark for? Is this a typo, or is this syntax that I've never seen before?

**Comment by cjpatton** [comment]
Hey @seanturner, thanks for pointing these out. Could you add another commit to this PR changing "public_key <0..2^16-1>" to "public_key <1..2^24-1>"?

File: draft-ietf-tls-subcerts.md (hunk 1 above)
Comment (seanturner) [reply draft-ietf-tls-subcerts.md]: {{?RFCXXXXX}} will insert an informative reference and {{!RFCXXXX} will insert a normative reference.  Learned that from martinthomson.

Comment (cjpatton) [reply draft-ietf-tls-subcerts.md]: Oh neat. Thanks for the tip!

**Comment by cjpatton** [comment]
The public key name and type have been changed in #18, as @seanturner suggested on the mailing list.

**Comment by seanturner** [comment]
@cjpatton beat me to it!

**Comment by siyengar** [comment]
oh looks like the travis build is failing for some reason

**Comment by seanturner** [comment]
@siyengar I have no idea why pushing that worked ...

**Comment by siyengar** [comment]
awesomesauce. it's probably some temporary failure while trying to fetch the new references or something

**Pull request merged**